psalm-baseline.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="5.15.0@5c774aca4746caf3d239d9c8cadb9f882ca29352">
+  <file src="src/AuraRouter.php">
+    <PossiblyUnusedMethod>
+      <code>__construct</code>
+    </PossiblyUnusedMethod>
+  </file>
+  <file src="src/AuraRouter/ConfigProvider.php">
+    <UndefinedClass>
+      <code>\Zend\Expressive\Router\AuraRouter</code>
+      <code>\Zend\Expressive\Router\RouterInterface</code>
+    </UndefinedClass>
```

Comment:
These can be changed to strings, i.e. 'Zend\Whatever' as opposed to `Zend\Whatever::class` - but it matters little for now 👍